Library/TimerLabelTests.swift

```diff
@@ -0,0 +1,83 @@
+import XCTest
+import Library
+
+final class TimerLabelTests: XCTestCase {
+
+    private weak var timer: Timer!
+
+    override func setUp() {
+        let mockScheduled = { [weak self] (interval: TimeInterval, repeats: Bool, block: @escaping (Timer) -> Void) -> Timer in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/7d4b7a47f520e10c677d21891d119ba0b38f461e/Library/TimerLabelTests.swift#L9"></span>Line should be 120 characters or less: currently 127 characters<br />
<code>line_length</code> <code>TimerLabelTests.swift:9</code></td>
    </tr>
  </tbody>
</table>